examples/images_contours_and_fields/streamplot_demo.py

```diff
@@ -0,0 +1,82 @@
+"""
+==========
+Streamplot
+==========
+
+A stream plot, or stream line plot, is used to display 2D vector fields. This
+example shows a few features of the stream plot function:
```

Comment:
I think generally, "streamline" is a word and "streamplot" is not, _unless_ you're referring to the function itself (as in this case.)